env/WCOSS2.env

```diff
@@ -13,6 +13,10 @@ step=$1
 export launcher="mpiexec -l"
 export mpmd_opt="--cpu-bind verbose,core cfp"
 
+# Add path to GDASApp libraries
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$HOMEgfs/sorc/gdas.cd/build/lib"
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/opt/cray/pe/mpich/8.1.19/ofi/intel/19.0/lib"
```

Comment:
Why not place this in `load_ufsda_modules.sh` since only GDASapp needs this.

Reply:
I think this is hopefully temporary @RussTreadon-NOAA I know is in contact with NCO/GDIT about some issues related to build and runtime with WCOSS

Reply:
We should remove these lines from `env/WCOSS2.env`.  These lines are already in `ush/load_ufsda_modules.sh`

RFC 13644 (implemented this week) removes the need for
```
export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/opt/cray/pe/mpich/8.1.19/ofi/intel/19.0/lib"
```
in `load_ufsda_modules.sh`.  We still need
```
export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$HOMEgfs/sorc/gdas.cd/build/lib"
```
in `load_ufsda_modules.sh`.

Reply:
thanks @RussTreadon-NOAA , done at dc36516